### PR TITLE
Medical Treatment - Suture

### PR DESCRIPTION
**Pull request by BrettMayson**
**When merged this pull request will:**
- [x] Add logic for sutures
- [x] A model and ui picture

Intended as an alternative to #8424
 
~~I've have yet to make a model for Arma 3, not sure about the process. If anyone else wants to make one it would be greatly appreciated, otherwise it may be a bit before I get around to figuring it out~~

### CODE REVIEW

**Comment by Walthzer** [comment]
For a sutures model, just a small package with one of them?
Capture

and/or a box that expands into packages when one gets used from it? 
15824ET-3967902197

**Comment by Walthzer** [comment]
Textures are a little iffy but how about these? 
result

**Comment by BrettMayson** [comment]
Looks good to me, I'd pick a less aggressive color though

**Comment by kymckay** [comment]
Would be good to colour match the green from the image above

**Comment by Walthzer** [comment]
Does look better yeah, good call. Trying to figure out a good UI picture now.
result

**Comment by Walthzer** [comment]
If I just take the model, this is the best it would be. Can't get legible text within a reasonable resolution. Otherwise I'd say we display it in the UI as thread with a needle? 
result

**Comment by BrettMayson** [comment]
I'd use a thread and needle yeah

**Comment by Walthzer** [comment]
Tried some different designs and this is what looks the "best", fitting a suture into the UI is awkward :P. Currently think the "Suture" picture is the better one. 

ui_pictures

**Comment by Walthzer** [comment]
Can't seems to create a PR onto your fork Brett. Probably doing something wrong. How would I go about "appending" to this PR from my end?

**Comment by PabstMirror** [comment]
@Walthzer 
https://github.com/BrettMayson/ACE3/compare/consume_stitches...Walthzer:ACE3:consume_stitches
post a link to your pr back here if you make one because it won't show up on this repo

**Comment by Walthzer** [comment]
Wierd, the GitHub Interface didn't allow me to select Brett from the dropdown, huh. But that link did the trick Pabst. Created a PR for Brett to add the models to the branch: [the PR](https://github.com/BrettMayson/ACE3/pull/2#issue-1361018455)

**Comment by BrettMayson** [comment]
Anyone know what the issue here is?

**Comment by Walthzer** [comment]
> Anyone know what the issue here is?

Sadly I do, I included the wrong models. They are trying to load textures files from the wrong path. Ugh. Will open a fix in a few.

**Comment by Walthzer** [comment]
@BrettMayson Opened a [PR](https://github.com/BrettMayson/ACE3/pull/3) that *should* fix the paths.

**Comment by BrettMayson** [comment]
Used this in an MP mission tonight, worked great and models are nice. Thanks @Walthzer for the contribution!

**Comment by BrettMayson** [comment]
Probably going to want to reduce the mass of the surgical kit, forgot to mention on my last post that we set it to `10`. Seems to be good so far, but we could go slightly more.

**Comment by BrettMayson** [comment]
We are getting `Cannot load texture z\wfar\addons\axe\data\suture_nohq.paa`

@Walthzer

**Comment by Walthzer** [comment]
That issue was due to the wrong model being copied over. I just checked the code again I am quite sure I had fixed it. Where is the error from?

**Review by veteran29 — CHANGES_REQUESTED** [review]

File: addons/medical_treatment/functions/fnc_surgicalKitStart.sqf [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ * None
+ *
+ * Example:
+ * [player, cursorObject] call ace_medical_treatment_fnc_surgicalKitStart
+ *
+ * Public: No
+ */
+
+if (GVAR(consumeSurgicalKit) == 2) then {
+    ([_medic, _patient, ["ACE_suture"]] call FUNC(useItem));
```
[the remark this answers is not in the document]
Comment (BrettMayson) [reply addons/medical_treatment/functions/fnc_surgicalKitStart.sqf:20]: Have my medics been using free suture's without reporting it as a bug :|

**Review by veteran29 — APPROVED** [review]
LGTM

File: addons/medical_treatment/ACE_Medical_Treatment_Actions.hpp [hunk 2]
```diff
@@ -286,6 +286,7 @@ class GVAR(actions) {
         treatmentTime = QFUNC(getStitchTime);
         condition = QFUNC(canStitch);
         callbackSuccess = "";
+        callbackStart = QFUNC(surgicalKitStart);
         callbackProgress = QFUNC(surgicalKitProgress);
         consumeItem = QGVAR(consumeSurgicalKit);
```
Comment (PabstMirror) [inline addons/medical_treatment/ACE_Medical_Treatment_Actions.hpp:291]: not sure if needed, but might help to add a little note about how it works (not like most bools)
```suggestion
        consumeItem = QGVAR(consumeSurgicalKit); // setting can be 0,1,2 - only 1 will consume items[]
```

**Review by PabstMirror — APPROVED** [review]

**Pull request merged**